### PR TITLE
Upgrade to YNH 4.3

### PR DESCRIPTION
**Pull request by ericgaspar**
- Add auto updater
- upgrade to YNH 4.3

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ajbura** [comment]
Looks like disclaimer file is little outdated. As per #3 it now support installation in subdirectories.

**Comment by ericgaspar** [comment]
Yes, you are right, I reverted the changes on path_url.
I still have problems to connect to the Matrix server with this app... I get `Looks like you have a lot of stuff to heat up!`for ever.
Are you able to connect?

**Comment by ericgaspar** [comment]
!testme

**Comment by ajbura** [comment]
Firefox has some issues with indexedDB in incognito, access to which is required by app.

**Comment by Salamandar** [comment]
> Looks like disclaimer file is little outdated. As per #3 it now support installation in subdirectories.

Yes, I just fixed that, thanks :D

File: conf/nginx.conf [hunk 1]
```diff
@@ -4,11 +4,6 @@ location __PATH__/ {
   # Path to source
   alias __FINALPATH__/;
 
-  # Force usage of https
-  if ($scheme = http) {
-    rewrite ^ https://$server_name$request_uri? permanent;
-  }
-
```
Comment (Salamandar) [inline conf/nginx.conf:11]: Why remove this section ? Is it now done at a whole-domain level ?

File: manifest.json [hunk 2]
```diff
@@ -22,7 +22,7 @@
         "email": "felix@piedallu.me"
     },
     "requirements": {
-        "yunohost": ">= 4.1.3"
+        "yunohost": ">= 4.3.0"
```
Comment (Salamandar) [inline manifest.json:25]: What technically requires this bump ?

File: conf/nginx.conf (hunk 1 above)
Comment (ericgaspar) [reply conf/nginx.conf:11]: NGINX HTTPS now part of the core

File: manifest.json (hunk 2 above)
Comment (ericgaspar) [reply manifest.json:25]: NGINX HTTPS part

File: conf/nginx.conf (hunk 1 above)
Comment (Salamandar) [reply conf/nginx.conf:11]: OK !

File: manifest.json (hunk 2 above)
Comment (Salamandar) [reply manifest.json:25]: Got it.

**Pull request merged**